chain/chain/src/resharding.rs

```diff
@@ -327,7 +354,7 @@ impl Chain {
     fn initialize_flat_storage(
         &self,
         prev_hash: &CryptoHash,
-        child_shard_uids: &[&ShardUId],
+        child_shard_uids: &Vec<ShardUId>,
```

Comment:
Didn't we have some convention saying we shouldn't be passing Vecs as params but only spans of vecs (the &[...])? @pugachAG

Reply:
Let me try, it does seem more "rusty".